crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1281,6 +1321,7 @@ impl<'db> Type<'db> {
             Type::Tuple(_) => KnownClass::Tuple.to_class(db),
             // TODO not accurate if there's a custom metaclass...
             Type::ClassLiteral(_) => KnownClass::Type.to_class(db),
+            Type::Type(_) => Type::Todo,
```

Comment:
This is too meta for me. Should this be `Type::Type('type')`?

Reply:
> Should this be `Type::Type('type')`?

yes, I think so

Reply:
For now, sure. But with a TODO comment. Because with Charlie's metaclass PR, it should become something like:
```suggestion
            Type::Type(class) => Type::Type(class.class.metaclass(db).expect_literal_class())
```

but with more proper handling of odd cases and less expect

Reply:
I tried something, but I'm not sure what kind of "odd cases" would need to be handled.